docs/source/array-slicing.rst

```diff
@@ -7,15 +7,19 @@ supports the following:
 *  Slicing by integers and slices ``x[0, :5]``
 *  Slicing by lists/arrays of integers  ``x[[1, 2, 4]]``
 *  Slicing by lists/arrays of booleans ``x[[False, True, True, False, True]]``
-*  Slicing one ``dask.array`` with another ``x[x > 0]``
+*  Slicing one `~dask.array.Array` with a `~dask.array.Array` of bools ``x[x > 0]``
+*  Slicing one `~dask.array.Array` with a zero or one-dimensional `~dask.array.Array`
+   of ints ``x[x.map_blocks(np.argsort)]``
```

Comment:
This is a problematic example as argsort won't behave as expected if x is chunked on the axis it operates. Should be replaced with argtopk  as soon as it is merged (#3405).